### ISSUE TITLE
Communicate zoom radius for Veterans using screenreaders

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#22649]
## Issue Description
Veterans using assistive tech to interact with the map need to know what changes have occurred on the map when the zoom buttons have been used. @AngelaFowler82 recommends announcing a zoom radius for screen readers (rather than the "zooming in, level X" functionality which was implemented in #17963. 

## Tasks
- [ ] Determine the amount of radius change which occurs each time the zoom button is clicked by using [this chart ](https://docs.mapbox.com/help/glossary/zoom-level/) to compute meters per pixel of the current zoom level, then multiply by the current width of the map (which depends on the screen size) to compute the current search radius.
- [ ] Ensure context changes are communicated, accessible to screen readers.
- [ ]  Add environment check to support testing in staging.
- [ ] Add an issue to remove the environment check 

## Acceptance Criteria
The following should be validated with screen reader and keyboard:

- [ ] Each time a Veteran using assistive tech clicks the zoom in button, the screenreader announces the level of radius change which has occurred on the map
- [ ]  Each time a Veteran using assistive tech clicks the zoom out button, the screenreader announces the level of radius change which has occurred on the map. that the map is "zooming out"

**Comment by mmiddaugh** [comment]
Screenreader announcement in miles was added to #22960
